koji/collection-mash-split.py

```diff
@@ -391,17 +391,10 @@ def main():
             git_tag = "rpm/{}".format(version)
 
         CONFIG["comps_path"] = "/mnt/koji/mash/comps/foreman"
-        if collection == 'foreman' and LooseVersion(version) <= LooseVersion('2.1'):
-            CONFIG["extras_baseloc"] = "%(tag)s:extras"
-            CONFIG["extras_path"] = "/mnt/koji/mash/extras/foreman"
-            extras = ["extras-foreman-rhel7"]
-        else:
-            extras = []
+        extras = []
 
         mashes = [MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7", extras)]
-
-        if LooseVersion(version) > LooseVersion('2.0'):
-            mashes.append(MashConfig(collection, version, "el8", "el8", "RHEL/8"))
+        mashes.append(MashConfig(collection, version, "el8", "el8", "RHEL/8"))
```

Comment:
instead of `.append`, you now could just add it in the real `mashes` definition the line above?